openedx/features/colaraz_features/api/urls.py

```diff
@@ -17,5 +17,5 @@
     url(r'^notifications/(?P<api_method>\D+)', NotificationHandlerApiView.as_view(), name="notifications_handler"),
     url(r'^job-alerts/(?P<api_method>\D+)', JobAlertsHandlerApiView.as_view(), name="job_alerts_handler"),
     url(r'^course/{}'.format(settings.COURSE_ID_PATTERN), CourseOutlineView.as_view(), name="course_outline"),
-    url(r'^xblock/{}$'.format(settings.USAGE_KEY_PATTERN), CourseXBlockApi.as_view(), name="course_xblock"),
+    url(r'^xblock/{}/$'.format(settings.USAGE_KEY_PATTERN), CourseXBlockApi.as_view(), name="course_xblock"),
```

Comment:
Why do we need this change? Will this affect Open edX's remaining functionalities?

Reply:
No it would change the open edX's remaining functionalities. As I have developed this api recently. just got new requirement by colaraz team.